SeaPkg/Library/BasePeCoffLibNegative/BasePeCoffLibNegative.c

```diff
@@ -981,6 +981,21 @@ PeCoffImageDiffValidation (
           NextImageValidationEntryHdr = (IMAGE_VALIDATION_ENTRY_HEADER *)(ImageValidationEntrySelfRef + 1);
         }
 
+        break;
+      case IMAGE_VALIDATION_ENTRY_TYPE_POINTER:
+        if (ImageValidationEntryHdr->Size >= sizeof (UINTN)) {

```

Comment:
```suggestion
        if (ImageValidationEntryHdr->Size > sizeof (UINTN)) {

```

I believe this should just be greater than.

Reply:
Funny I was so concerned about the pointer logic, and I messed up on this 🗡️

Reply:
Updated, thank you!